flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why limiting the architecture?

Reply:
I've not tested building for aarch64 but no reason we can't try it.